worlds/witness/__init__.py

```diff
@@ -67,7 +67,7 @@ def _get_slot_data(self):
             'progressive_item_lists': self.items.MULTI_LISTS_BY_CODE,
             'obelisk_side_id_to_EPs': self.static_logic.OBELISK_SIDE_ID_TO_EP_HEXES,
             'precompleted_puzzles': {int(h, 16) for h in self.player_logic.PRECOMPLETED_LOCATIONS},
-            'ep_to_name': self.static_logic.EP_ID_TO_NAME,
+            'entity_to_name': self.static_logic.ENTITY_ID_TO_NAME,
```

Comment:
since you removed something from slot data - does the old client correctly reject new seeds and vice versa?

Reply:
Old clients will not crash but maybe behave weirdly on new seeds, but that is to be expected. (Required versions have been changed since the last full/stable release of both AP and the client. Older prereleases of the client might behave strangely. I feel like that's fine though)

New client can handle both old and new seeds - It checks for the existence of both the old field and the new field and handles them appropriately.